modules/telco-upgrade-performing-the-second-y-stream-update.adoc

```diff
@@ -0,0 +1,85 @@
+// Module included in the following assemblies:
+//
+// * edge_computing/day_2_core_cnf_clusters/upgrading/telco-upgrade-completing-the-upgrade.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="telco-upgrade-performing-the-second-y-stream-update_{context}"]
+= Performing the second y-stream update
+
+After completing the first y-stream update, you need to upgrade the y-stream control plane version to the new EUS version.
```

Comment:
Adding onto my confusion about skipping sections, I think it should be made clearer in each of these sections that can be skipped, that they can be skipped for z-stream/y-stream. And are only for EUS/Control Plane Only.

Or... I Know it would be somewhat duplicative but maybe they should be separate assemblies. Performing the update for EUS updates, and performing the update for non-EUS updates.

Reply:
Updated with discrete assemblies. Works much better now.